File: Java/ReverseString.java
class ReverseString {
    
    public String reverseString(String s) {
        
        char[] arr = s.toCharArray();
        int start = 0;
        int end = arr.length - 1;
        while(start < end) {
            char temp = arr[start];
            arr[start] = arr[end];
            arr[end] = temp;
            start++;
            end--;
        }
        
        return new String(arr);
        
        
    }
    
}

File: C++/ReverseString.cpp
class Solution {
public:
    void reverseString(vector<char>& s) 
    {
        char temp;
        int end = s.size() - 1;
        int start = 0;
        while(start < end)
        {
            temp = s[start];
            s[start] = s[end];
            s[end] = temp;
            ++start;
            --end;
        }
    }
};